docs/Users/Tutorials/Writing_Bears.rst

```diff
@@ -188,3 +188,65 @@ yield them and write the method as a generator.
     about how bears will look in the future, please read up on
     https://github.com/coala-analyzer/coala/issues/725 or ask us on
     https://gitter.im/coala-analyzer/coala.
+
+Bears With Other Bears as Dependencies
+--------------------------------------
+
+So we've got a result, but what if we need our Bear to depend on results from
+a different Bear?
+
+Well coala has an efficient dependency management system
+that would run the other Bear before your Bear and get its results for you.
+All you need to do is to tell coala which Bear/s you want to run before your
+Bear.
+
+So lets see how we tell coala which Bears to run before yours
+
+.. code:: python
+
+    from coalib.bears.LocalBear import LocalBear
+    from bears.somePathTo.OtherBear import OtherBear
+
+    class DependancyBear(LocalBear):
```

Comment:
Also, maybe name this as `MainBear` or `DependentBear` as this is the one that is dependent and not the dependency